validation/src/test/scala/hmda/validation/engine/lar/syntactical/LarSyntacticalEngineSpec.scala

```diff
@@ -0,0 +1,22 @@
+package hmda.validation.engine.lar.syntactical
+
+import hmda.parser.fi.lar.LarGenerators
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+class LarSyntacticalEngineSpec
+    extends PropSpec
+    with PropertyChecks
+    with MustMatchers
+    with LarGenerators
+    with LarSyntacticalEngine {
+
+  property("A LAR must pass syntactical checks") {
+    forAll(larGen) { lar =>
+      whenever(lar.id == 2) {
+        checkSyntactical(lar).isSuccess mustBe true
```

Comment:
We'll need a test for `checkSyntacticalIterable` too. If it's not feasible to add it now, we could add an issue for it, or add a note to #276.